### PR TITLE
Magento on branch

### PR DESCRIPTION
**Pull request by GrandLTU**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -24,8 +26,9 @@
         "ongr/content-bundle": "~0.1",
         "ongr/pager-bundle": "dev-master",
         "ongr/router-bundle": "~0.2",
-        "ongr/ssi-bundle": "v0.3.0"
+        "ongr/ssi-bundle": "v0.3.0",
 
+        "ongr/magento-connector-bundle": "dev-master"
```
Comment (trylika) [inline composer.json]: why dev-master? maybe we should use more specific version?

File: src/ONGR/DemoMagentoBundle/DependencyInjection/Configuration.php [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function getConfigTreeBuilder()
+    {
+        $treeBuilder = new TreeBuilder();
+        $rootNode = $treeBuilder->root('ongr_demo_magento');
+
+        // Here you should define the parameters that are allowed to
```
Comment (trylika) [inline src/ONGR/DemoMagentoBundle/DependencyInjection/Configuration.php]: useless comment

File: composer.json (hunk 1 above)
Comment (lmikelionis) [inline composer.json]: You are advising to version this Bundle? Is it not to soon?

Comment (trylika) [inline composer.json]: it's never too soon for 0.01 alpha :D

**Pull request merged**